amy/static/scss/custom_bootstrap.scss

```diff
@@ -12,12 +15,13 @@ $gray-300: #dee2e6;
 $gray-400: #ced4da;
 $gray-500: #adb5bd;
 $gray-600: #6c757d;
+$gray-650: #5a636a; // custom
```

Comment:
Could you perhaps use some calculate function to get this value, for example:
```scss
$gray-650: hsl(208, 7%, 46% - 8%);  // hsl values for $gray-600
```
This is an example, as the value you entered for `$gray-650` is `hsl(206, 8%, 38%)` - very similar to what I put above, but not quite the same.

...
After some thinking I decided to add here that using `hsl` or any other color function only makes sense if you modified another color by e.g. increasing or decreasing individual component (like lightness or saturation, or hue). If there's too many components changed then it doesn't really make any sense.

Reply:
I switched to hsl, as it's _mainly_ a lightness change (difference of 2 in the other components between `gray-600` and `gray-700`).